### PR TITLE
test(invity): Integration Test Exchange

### PR DESCRIPTION
**Pull request by FelixVermont**
Successful scenario of the Exchange functionality within the Trade section.

### CODE REVIEW

File: packages/integration-tests/projects/suite-web/tests/coinmarket/exchange.test.ts [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        cy.getTestElement('@settings/wallet/network/eth').click();
+        cy.getTestElement('@settings/menu/close').click();
+
+        // Goes to Exchange
+        cy.getTestElement('@suite/menu/wallet-index').click();
+        cy.getTestElement('@account-menu/regtest/normal/0/label').click();
+        cy.getTestElement('@wallet/menu/wallet-coinmarket-buy').click();
+        cy.getTestElement('@coinmarket/menu/wallet-coinmarket-exchange').click();
+    });
+
```
Comment (ondracja) [inline packages/integration-tests/projects/suite-web/tests/coinmarket/exchange.test.ts]: Add test case description via JSDoc comment, for reference see eg. `add-account-types.test.ts`

File: packages/integration-tests/projects/suite-web/tests/coinmarket/exchange.test.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        // Verifies amounts, currencies and providers
+        cy.get('[class*="CoinmarketExchangeOfferInfo__Wrapper"]')
+            .should('exist')
+            .then(wrapper => {
+                cy.wrap(wrapper)
+                    .find('[class*="FormattedCryptoAmount__Value"]')
+                    .first()
+                    .invoke('text')
+                    .should('be.equal', '0.005');
```
Comment (ondracja) [inline packages/integration-tests/projects/suite-web/tests/coinmarket/exchange.test.ts]: Move all inputed values into an object at the beginning of the test and use references withing the test itself. It improves readability and helps with reusage. For reference, check eg `bug-report-form.test.ts`

File: packages/integration-tests/projects/suite-web/cypress.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
 {
     "integrationFolder": "./tests",
+    "baseUrl": "http://localhost:8000",
```
Comment (ondracja) [inline packages/integration-tests/projects/suite-web/cypress.json]: This parameter is set via docker in the CI so it shouldn't be set here.

File: packages/integration-tests/projects/suite-web/tests/coinmarket/exchange.test.ts [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+     * 1. Go to Accounts/REGTEST account/Trade/Exchange
+     * 2. Check whether all input windows are empty and crypto input is set on REGTEST
+     * 3. Exchange 0.005REGTEST for ETH with custom fee of 1 sat
+     * 4. Verifies the mocked offers are all fully and correctly displayed
+     * 5. Pick one offer
+     * 6. Verifies the amounts, currencies, addresses and providers are all in accordance with the mock
+     * 7. Confirms the transaction and verifies the same information in the modal
+     * 8. Return back to the Exchange tab in Trezor Suite
+     */
+
```
Comment (ondracja) [inline packages/integration-tests/projects/suite-web/tests/coinmarket/exchange.test.ts]: Remove the blank line; the test case documentation should undoubtly belong to the `it`.

**Comment by mroz22** [comment]
I suggest you rebase on develop as soon as possible. There were many changes in e2e tests setup

**Comment by martinboehm** [comment]
@mroz22 it is rebased to today's develop.

**Review by ondracja — APPROVED** [review]

**Pull request merged**